scripts/syntactics/index.rst

```diff
@@ -0,0 +1,78 @@
+Deep Biaffine Dependency Parser
+---------------------------------
+
+This package is an implementation of `Deep Biaffine Attention for Neural Dependency Parsing <https://arxiv.org/pdf/1611.01734.pdf>`_ proposed by Dozat and Manning (2016), with SOTA accuracy.
+
+Train
+^^^^^^^^^^^^^
+
+As the Penn Treebank dataset (PTB) is proprietary, we are unable to distribute it.
+If you have a legal copy, please place it in ``tests/data/biaffine/ptb``, use this `pre-processing script <https://github.com/hankcs/TreebankPreprocessing>`_ to convert it into conllx format.
+Then download `glove embeddings <http://nlp.stanford.edu/data/glove.6B.zip>`_ into ``tests/data/biaffine/embedding`` and unzip it.
+The tree view of data folder should be as follows.
+
+.. code-block:: console
+
+	$ tree tests/data/biaffine
+	tests/data/biaffine
+	├── embedding
+	│   └── glove.6B.100d.txt
+	└── ptb
+		├── dev.conllx
+		├── test.conllx
+		└── train.conllx
+
+Then Run the following code to train the biaffine model.
+
+.. code-block:: python
+
+    parser = DepParser()
+    parser.train(train_file='tests/data/biaffine/ptb/train.conllx',
+                 dev_file='tests/data/biaffine/ptb/dev.conllx',
+                 test_file='tests/data/biaffine/ptb/test.conllx', save_dir='tests/data/biaffine/model',
+                 pretrained_embeddings_file='tests/data/biaffine/embedding/glove.6B.100d.txt')
```

Comment:
This is available in gluonnlp API as `gluonnlp.embedding.create('glove', 'glove.6B.100d')`

Reply:
Yes, got it.